irohad/main/application.hpp

```diff
@@ -22,22 +22,28 @@
 #include <ametsuchi/ametsuchi.hpp>
 #include <dao/dao.hpp>
 
-namespace main {
+namespace iroha {
 
   /**
    * Contains instances of global services and shared providers
    */
+  using namespace iroha;
   class Irohad {
```

Comment:
Currently it is not used, since all objects are initialized in `main()`. It should contain shared objects (such as ametsuchi, crypto/hash providers) in the future, since their initialization might be complex.